src/Engine/Renderer/RenderObject/RenderObject.hpp

```diff
@@ -160,7 +155,7 @@ class RA_ENGINE_API RenderObject final : public Core::Utils::IndexedObject
      * @param viewParams viewing parameters for this rendering
      * @param passname RenderTechnique pass name
      */
-    virtual void render( const RenderParameters& lightParams,
+    void render( const RenderParameters& lightParams,
```

Comment:
Code style is not respected here, the following lines should be indented accordingly to this change